osc-dm-product-srv/src/utl/openaillm.py

```diff
@@ -0,0 +1,36 @@
+
+import os
+from openai import OpenAI
+
+class OpenAIClient:
+
+    def __init__(self, model: str = "gpt-3.5-turbo",):
+        self.client = OpenAI(
+            # This is the default and can be omitted
+            api_key=os.environ.get("OPENAI_API_KEY")
+        )
```

Comment:
recommend checking for existance, and throwing an exception if key is not available.

Reply:
updated so exception is raised if the env variable does not exist